frontend/javascripts/oxalis/geometries/skeleton.js

```diff
@@ -552,40 +547,6 @@ class Skeleton {
   getTreeRGBA(color: Vector3, isVisible: boolean) {
     return color.concat(isVisible ? 1 : 0);
   }
-
-  /**
-   * Calculates a resizing factor for the active node's radius every time the
-   * active node id changes. In essence this animates the node's radius to grow/shrink a little.
-   */
-  async startNodeHighlightAnimation() {
-    const normal = 1.0;
-    const highlighted = 2.0;
-
-    await this.animateNodeScale(normal, highlighted);
-    await this.animateNodeScale(highlighted, normal);
-  }
-
-  /**
-   * Calculates a resizing factor for the active node's radius every time the
-   * active node id changes. In essence this animates the node's radius to grow/shrink a little.
-   */
-  animateNodeScale(from: number, to: number): Promise<void> {
-    return new Promise((resolve, reject) => {
-      const setScaleFactor = scale => {
-        this.nodes.material.uniforms.activeNodeScaleFactor.value = scale;
```

Comment:
This uniform can be removed now, it's no longer changed anywhere and its default value is 1.

Reply:
Good point, thanks for looking that closely!